client/my-sites/plans/current-plan/jetpack-checklist/header.js

```diff
@@ -20,13 +20,21 @@ const JetpackChecklistHeader = ( { isPaidPlan, translate } ) => (
 		/>
 		<div className="jetpack-checklist__header-content">
 			<CardHeading>
-				{ translate( "Let's start by securing your site with a few essential security features" ) }
+				{ isCompleted ? (
+					<>
+						{ translate( 'Congratulations' ) }
+						<p>
+							{ translate(
+								"You've completed setting up all of the essential security and performance features. Going forward, you can access these features in the Settings sidebar menu."
+							) }
+						</p>
+					</>
+				) : (
+					translate(
+						"Let's start by securing your site with a few essential security and performance features"
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Let's start by securing your site with a few essential security features' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).